.github/workflows/macos.yml

```diff
@@ -38,30 +38,43 @@ jobs:
   build_and_test:
     strategy:
       matrix:
-        os: [macos-13, macos-14]
-        compiler: [g++-12, g++-13, g++-14, clang++]
+        os: [macos-13, macos-14, macos-15]
+        architecture: [x64, arm64]
+        compiler: [g++-13, g++-14, clang++]
         include:
           - os: macos-13
             architecture: x64
           - os: macos-14
             architecture: arm64
+          - os: macos-15
+            architecture: arm64
```

Comment:
Now that you explicitly include the architectures in the matrix above, these statements are redundant since all architectures are matched against all operating systems by default. It is then left to exclude the combinations you don't want. I'd prefer the other way around as it was before because we only need these three includes then and no excludes for the architectures.

Reply:
ah, great! Thanks!